curriculum/tools/README.md

```diff
@@ -26,6 +26,17 @@ A one-off script that will automatically add a specified numbers of steps at a s
   npm run create-empty-steps start=X num=Y # where X is the starting step number and Y is the number of steps to create.
   ```
 
+## [create-step-between.js](create-step-between.js)
+A one-off script that automatically adds a new step between two existing consecutive steps. The challenge seed code will use the existing starting step's challenge seed code with the editable region markers (ERMs) removed.
+
+**Note:** This script also runs [reorder-steps.js](reorder-steps.js).
+
+### How to run script:
+1. Change to the directory of the project.
+2. Run the following npm command:
+  ```bash
+  npm run create-step-between start=X end=Y # where X is the starting step number and Y is the following step number.
+
```

Comment:
````suggestion
  ```bash
  npm run create-step-between start=X end=Y # where X is the starting step number and Y is the following step number.
  ```
````